### ISSUE TITLE
Include microbit into Sugarizer

### ISSUE THREAD

**Issue by llaske** [llaske/sugarizer#562]
[Microbit](https://makecode.microbit.org/) is a popular programming environment based on the Microsoft Makecode tool that  can enable offline mode.

![image](https://user-images.githubusercontent.com/981841/72210636-9f580d80-34be-11ea-9a03-9dae2662268f.png)

The idea is to explore the possibility to include Microbit as a Sugarizer activity, like Scratch.

To do that, it's necessary to:
* [ ] Extract a working offline version of Microbit that could run in a browser without Internet access
* [ ] Create a sample Sugarizer activity following Sugarizer developement activity tutorial
* [ ] Integrate Microbit into the sample activity
* [ ] Update Microbit to store/load its content into Sugarizer datastore

**Comment by sarthak-g** [comment]
working on this

**Comment by sarthak-g** [comment]
I've created sample activity and successfully extracted a offline version of microbit. See below:
![microbitLOCALss](https://user-images.githubusercontent.com/42293606/72663283-c5c5fd80-3a16-11ea-90b8-0ab1b48e8ba0.PNG)
I tried but not able to understand how to integrate it with sugarizer as microsoft uses "pxt" to run microbit.
We have to run "pxt serve"in order to run it locally.
Can you suggest how I can approach?

**Comment by llaske** [comment]
Hmmm. When I said "offline" my idea was to have microbit that fully run in the browser (no call to a server, including a local server). Look like there is some work on this [here](https://makecode.microbit.org/offline). But may be it's not possible today.

**Comment by sarthak-g** [comment]
> 
> 
> But may be it's not possible today.
Yeah, it exist only as standalone desktop application in offline.

**Comment by AndreaGon** [comment]
@sarthak-g Are you still working on this? Can I look into this as well?

**Comment by sarthak-g** [comment]
Yes, you can.
I will love to know if you find anything relevant.

**Comment by shotaro27** [comment]
Hello. I'm Shotaro27.

Offline version of Microbit was built already, so I can't use it as Web application.
[Pxt-microbit](https://github.com/shotaro27/pxt-microbit) is not able to run offline, and it need a lot of preparation.
So I couldn't find offline version application for Sugarizer.

I tried to create a Sugarizer activity following Sugarizer developement activity tutorial.

![スクリーンショット (207)](https://user-images.githubusercontent.com/11911384/72820206-e78ee100-3cb1-11ea-8b42-a144e61c5261.png)
![スクリーンショット (208)](https://user-images.githubusercontent.com/11911384/72820207-e78ee100-3cb1-11ea-86b7-662a5867f6f1.png)

**Comment by sarthak-g** [comment]
> 
> 
> Offline version of Microbit was built already, so I can't use it as Web application.
> [Pxt-microbit](https://github.com/shotaro27/pxt-microbit) is not able to run offline, and it need a lot of preparation.
> So I couldn't find offline version application for Sugarizer.
> 

Hi Shotaro27,
I and @llaske  discussed regarding this above.
Actually idea is to get gui of microbit (like there in scratch).

**Comment by yashveeeeer** [comment]
Hi @llaske, I would like to work on this one , could you guide me a bit.

**Comment by llaske** [comment]
@yashveeeeer I don't see any progress on this side.
We can close the issue.
